everything_shell/getenv.c

```diff
@@ -0,0 +1,53 @@
+#include <unistd.h>
+#include <stdlib.h>
```

Comment:
This file does not make use of <stdlib.h>

Reply:
the environment file is in the stdlib

Reply:
i  just realized i didnt use the getenv function here